RELEASE_NOTES.md

```diff
@@ -1,5 +1,60 @@
 # Release Notes
 
+## 5.2.0
+### Features
+- Quadlet now has support for `.build` files, which allows images to be built by Quadlet and then used by Quadlet containers.
+- Quadlet `.container` files now support two new fields, `LogOpt` to specify container logging configuration and `StopSignal` to specify container stop signal ([#23050](https://github.com/containers/podman/issues/23050)).
+- Quadlet `.container` and `.pod` files now support a new field, `NetworkAlias`, to add network aliases.
+- Quadlet drop-in search paths have been expanded to include top-level type drop-ins (`container.d`, `pod.d`) and truncated unit drop-ins (`unit-.container.d`) ([#23158](https://github.com/containers/podman/issues/23158)).
+- Podman now supports a new command, `podman system check`, which will identify (and, if possible, correct) corruption within local container storage.
+- The `podman machine reset` command will now reset all providers available on the current operating system (e.g. ensuring that both HyperV and WSL `podman machine` VMs will be removed on Windows).
+
+### Changes
+- Podman now requires the new kernel mount API, introducing a dependency on Linux Kernel v5.2 or higher.
+- Quadlet `.image` units now have a dependency on `network-online.target` ([#21873](https://github.com/containers/podman/issues/21873)).
+- The `--device` option to `podman create` and `podman run` is no longer ignored when `--privileged` is also specified ([#23132](https://github.com/containers/podman/issues/23132)).
+- The `podman start` and `podman stop` commands no longer print the full ID of the pod started/stopped, but instead the user's input used to specify the pod (e.g. `podman pod start b` will print `b` instead of the pod's full ID) ([#22590](https://github.com/containers/podman/issues/22590)).
+- Virtual machines created by `podman machine` on Linux now use `virtiofs` instead of `9p` for mounting host filesystems. Existing mounts will be transparently changed on machine restart or recreation. This should improve performance and reliability of host mounts.
```

Comment:
this should point out that that means virtiofsd must be installed on the host and there must be symlink created to the proper location, ref https://github.com/containers/podman/commit/180cc6f86324161c641513f291a1d01434876518

Reply:
(but I guess I am fine to leave it for now and add it for the final release or next rc)